### PR TITLE
Remove old sims and refactor

### LINKED ISSUE #132: Remove old sim code
At some point the sim code was completely rewritten. We need to remove the old version which is still in the code base.

### PR DESCRIPTION
**Pull request by esheldon**
closes #132

### CODE REVIEW

File: descwl_shear_sims/artifacts/cosmic_rays.py [hunk 1]
```diff
@@ -1,44 +1,13 @@
-"""
-copy-paste from my (beckermr) personal code here
-https://github.com/beckermr/metadetect-coadding-sims/blob/master/coadd_mdetsims/masking.py
-"""
```
Comment (beckermr) [inline descwl_shear_sims/artifacts/cosmic_rays.py:4]: Can we please keep this provenance?

**Review by beckermr — CHANGES_REQUESTED** [review]
One comment. Thanks for doing this!

**Comment by esheldon** [comment]
I'm going to add some additional testing

**Comment by esheldon** [comment]
we need more tests but I'm ready to merge this

**Review by beckermr — APPROVED** [review]

**Pull request merged**